packages/smooth_app/lib/pages/onboarding/knowledge_panel_page_template.dart

```diff
@@ -149,30 +149,44 @@ class _KnowledgePanelPageTemplateState
       );
 
   List<Widget> _buildHintPopup() {
-    final Widget hintPopup = InkWell(
+    final Widget hintPopup = Card(
```

Comment:
We still need the whole tooltip to be closable by click. The cross icon is just to tell the user "btw you can close this tooltip" (but you can click anywhere).
Besides if you want an icon to be clickable, you're supposed to make it "big enough" (something like 48) and it would be too big for the tooltip.